Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl

```diff
@@ -272,7 +279,7 @@ ghost-role-information-medical-description = You are a medical professional, but
 ghost-role-information-cargo-name = Cargo
 ghost-role-information-cargo-description = You are part of a logistics mission, but seem to have found yourself in a strange situation...
 
-ghost-role-information-engineering-name = Engineering 
+ghost-role-information-engineering-name = Engineering
```

Comment:
Untouch

Reply:
Removed by accident when saving the file.